haoshiyou/app/pages/listings-tab/listing-detail.page.html

```diff
@@ -15,7 +15,9 @@
         </ion-item>
         <ion-item>
             <h2>{{listing.title}}</h2>
-            <p>{{listing.lastUpdated|timeFromNow}}</p>
+            <p>{{listing.lastUpdated|timeFromNow}}
+                <font color= "{{listing.type==0? '#32db64':'#387ef5'}}">{{listing.price==undefined? ' ':('$'+listing.price)}}</font>
```

Comment:
Consider use readable term like 
listing.type== ListingType.ROOMMATE_WANTED, instead of "listing.type==0"